packages/core/contracts/optimistic-asserter/implementation/OptimisticAsserter.sol

```diff
@@ -337,6 +337,19 @@ contract OptimisticAsserter is OptimisticAsserterInterface, Lockable, Ownable {
         return SovereignSecurityInterface(ss).isDisputeAllowed(assertionId, msg.sender);
     }
 
+    function _isIdentifierSupported(bytes32 identifier) internal returns (bool) {
```

Comment:
nit: Maybe the naming could be improved so it shows the function could be storing something?
`_validateAndCacheIdentifier` and `_validateAndCacheCurrency` maybe?